axum/src/extract/mod.rs

```diff
@@ -28,6 +28,8 @@ pub use self::{
     request_parts::{BodyStream, RawBody},
 };
 
+pub mod host;
```

Comment:
Since this module only contains one type I think we should make the module private and `pub use` the type from here. Like we do with other extractors.

Reply:
I've updated to use the same style, is that what you had in mind?

Reply:
Yes!